### ISSUE TITLE
Player binaries lack audio API selection

### ISSUE THREAD

**Issue by OPNA2608** [ValleyBell/libvgm#53]
`player` (and `vgmtest`) doesn't offer the user any choice w/r/t what audio API and device should be used, unlike `audiotest`. In certain scenarios, this makes it impossible to use the player.

Under Linux with ALSA, PulseAudio and libao support enabled, it defaults to using ALSA (which should ideally be the last option in a line of gracious fallbacks). Playback in my case will default to the incorrect audio device, leaving me with no sound and a blocked audio device. With my Pulse mixer open and occupying the underlying ALSA device, ALSA initialization and thus playback fails.

```
Loading VGM ...
Opening Audio Device ...
Using driver ALSA.
Opening Device 0 (default) ...
ALSA lib pcm_dmix.c:1090:(snd_pcm_dmix_open) unable to open slave
Dev Init Error: C0
Done.
```

**Comment by ValleyBell** [comment]
Hmm ... good point.
At least for the "player" program I could make it use the "last" option.

**Comment by ValleyBell** [comment]
fixed with commit e1b03bebbb7450a152d2a69599cb807d3f544772
